### PR TITLE
add a dropdown in logout button

### PR DESCRIPTION
**Pull request by khunshan-coaldev**
- [ ] in addition to logout add settings and reports buttons
- [ ] reports should only be accessed by super admin.

### CODE REVIEW

**Comment by khunshan-coaldev** [comment]
- Reports (Only sup[eradmin)
- Settings (Admin and super admin)

File: src/components/reports/Basic.vue [hunk 1]
```diff
@@ -0,0 +1,295 @@
+<template>
+  <div id="expense">
+    <div class="diff-shadow">
+      <div class="page-upper">
+        <ul class="nav nav-tabs">
+          <!-- <li :class="tab === 'order' ? 'active-li ab1 tab1' : 'ab1 tab1'" @click="onClickOrder()">
```
Comment (ammarhaider001) [inline src/components/reports/Basic.vue]: remove commented code.

File: src/components/reports/Basic.vue [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+            <span>
+              <strong>Operator Sales</strong>
+            </span>
+          </li> -->
+          <li :class="tab === 'report' ? 'active-li ab1 tab1' : 'ab1 tab1'" @click="onClickReport()">
+            <span>
+              <strong>Profit/Loss Report</strong>
+            </span>
+          </li>
+          <!-- <li :class="tab === 'StockStatement' ? 'active-li ab1 tab2' : 'ab1 tab2'" @click="onClickStockStatement()"
```
Comment (ammarhaider001) [inline src/components/reports/Basic.vue]: remove commented Code

File: src/components/reports/Report.vue [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+      <div class="flex-box">
+        <div class="date-container">
+          <DateRange @dateRangeChange="setRange"  />
+        </div>
+        <div class="b" >
+          <button class="btn btn-orange" @click="fetchAnalyticsBtn">Search</button>
+        </div>
+      </div>
+      </div>
+    <!-- commnet out -->
```
Comment (ammarhaider001) [inline src/components/reports/Report.vue]: remove commented code

**Review by ammarhaider001 — CHANGES_REQUESTED** [review]

**Review by ammarhaider001 — APPROVED** [review]

**Pull request merged**